README.md

```diff
@@ -13,6 +13,7 @@ its other dependencies to your GOPATH or GOBIN (usually `~/go/bin/`) with the
 go tool:
 
 ```console
+$ sudo apt install libpcap-dev 
```

Comment:
This is a distribution-specific change.

Reply:
See my other comment (https://github.com/hwipl/listnd/pull/2#issuecomment-1046643308)